modules/imaging_browser/jsx/ImagingBrowserDDT.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Created by dblader on 7/25/17.
+ */
+/* global DynamicDataTable */
+class ImagingBrowserDDT extends DynamicDataTable {
```

Comment:
There used to be a bigger page about it on React's website, but inheritance is generally considered an anti-pattern in React. See https://facebook.github.io/react/docs/composition-vs-inheritance.html for alternatives.

("At Facebook, we use React in thousands of components, and we haven't found any use cases where we would recommend creating component inheritance hierarchies.")